MANIFEST.in

```diff
@@ -1,3 +1,6 @@
-include AUTHORS
-include LICENSE
-include README
+prune .github
+prune tests
+prune utils
+exclude .coveragerc
+exclude .gitignore
+exclude README.md
```

Comment:
Previously, the version was hardcoded in `setup.py`.

To avoid this, it now uses https://github.com/pypa/setuptools_scm which takes the version number from the tag.

By default, it also includes all files into the sdist from Git. So we no longer need any `include` lines in the manifest, and we should explicitly prune or exclude files we don't want.

In other projects I normally include stuff that's also in Git. But I've tried to match the previous behaviour here. We can adjust this to how you like.